spec/yaml/swiftnav/sbp/signing.yaml

```diff
@@ -154,12 +154,16 @@ definitions:
             fill: u8
             size: 4
             desc: The last 4 bytes of the certificate's SHA-1 fingerprint
+        - n_signature_bytes:
+            type: u8
+            desc: >
+              Length of the signature in bytes.
         - signature:
             type: array
             fill: u8
-            size: 64
+            size_fn: n_signature_bytes
```

Comment:
I think this needs to be a fixed size of 73 bytes, the `n_signature_bytes` field will dictate how many bytes of the signature are valid and/or should be used.

Reply:
Ahh good catch